docs/contributing/local-development/Local_Linking.mdx

```diff
@@ -45,6 +46,24 @@ Now, you should be able to run ESLint in the local downstream repository as you
 To check that the local package is being used, consider adding a `console.log("Hello, world!");` to a file such as `./packages/eslint-plugin/dist/index.js` and making sure that log appears when linting the local downstream repository.
 :::
 
+## Testing Rules
+
+Now that you've linked the `@typescript-eslint/*` packages with your local downstream repository, the next step would be to include the rule on the local repository ESLint configuration file, e.g:
+
+```json
+{
+  "rules": {
+    "@typescript-eslint/your-awesome-rule": "error"
+  }
+  // etc
+}
+```
+
+After that, you just need to run your repository `lint` script.
```

Comment:
"Just" implies it's easy. But nothing is easy for everyone. 

```suggestion
After that, you need to run your repository `lint` script.
```

Reply:
Perfect, makes total sense.